assets/components/PersonalizeHomeModal.tsx

```diff
@@ -259,7 +261,7 @@ class PersonalizeHomeModal extends React.Component<IProps, IState> {
                                 {
                                     gettext(
                                         '{{size}} out of 6 topics selected',
-                                        {size: this.state.selectedTopicIds.length}
+                                        {size: this.state.selectedTopicIds.length || '0'}
```

Comment:
I think it'd be better if you used `??`

Reply:
found the problem why it didn't work, it was in the gettext